src/pages/index.js

```diff
@@ -60,6 +105,39 @@ function handlePreviewPicture(card) {
   previewImagePopup.open(card);
 }
 
+// DELETE CARD HANDLER
+
+function handleDeleteCardClick(card) {
+  deleteConfirmationPopup.open();
+  deleteConfirmationPopup.setDeleteConfirm(() => {
+    deleteConfirmationPopup.setDeleteLoading(true);
+    api
+      .deleteCard(card._id)
+      .then((result) => {
+        deleteConfirmationPopup.close();
+        card.handleDeleteCard(result);
+      })
+      .catch((err) => {
+        alert(`Error! ${err}`);
+      })
+      .finally(() => {
+        deleteConfirmationPopup.setDeleteLoading(false);
+      });
+  });
+}
+
+// LIKE HANDLER
+
+function handleLikeButtonClick(card) {
+  api
+    .setLike(card._id, card.isLiked())
+    .then((res) => {
+      card.handleLikeIcon(res);
+    })
+    .catch((err) => {
+      alert(`Error! ${err}`);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors